tests/e2e/flavors.js

```diff
@@ -1,7 +1,24 @@
+const ALL_FLAVORS = ['js', 'js-umd', 'react', 'vue'];
+
 module.exports = {
-  flavors: ['js', 'js-umd', 'react', 'vue'].filter(
```

Comment:
there's a small chance this is used in jsdoc type somewhere, do you mind checking? not sure if I changed it to string at a later point

Reply:
I looked at the current codebase and commits in the change history of this file, I could not find anything.